plugins/HtmLawed/class.htmlawed.plugin.php

```diff
@@ -97,7 +97,9 @@ public function Format($Html) {
          'Status' => 1,
       );
 
-      $Spec = 'object=-classid-type, -codebase; embed=type(oneof=application/x-shockwave-flash); a=class(noneof=Hijack|Dismiss|MorePager/nomatch=%pop[in|up|down]|flyout|ajax%i)';
+      $Spec = 'object=-classid-type, -codebase; embed=type(oneof=application/x-shockwave-flash); ';
+      //$Spec .= 'a=class(noneof=Hijack|Dismiss|MorePager/nomatch=%pop[in|up|down]|flyout|ajax%i); ';
+      $Spec .= 'a,span,div,p,li,i,b,strong,em,code,blockquote,dd,dt,img,pre,h1,h2,h3,h4,h5,h6,s,del,sup,sub,q,small,cite,ins=class(oneof=Quote|Spoiler|CodeBlock|Spoiled|P|ImageResized|AuthorAction|MeAction/match=%^(post-|Align)%i)';
```

Comment:
I think we should remove: li,i,b,strong,em,dd,dt,h1,h2,h3,h4,h5,h6,s,del,sup,sub,q,small,cite,ins.